### PR TITLE
[Fix - ADF 153]: Fix regresion of QTI spec

### PR DESCRIPTION
**Pull request by habemuscode**
**Issue**

https://oat-sa.atlassian.net/browse/ADF-153

**PR that introduce the regression**

https://github.com/oat-sa/extension-tao-itemqti/pull/1656

**Documentation**

https://oat-sa.atlassian.net/wiki/spaces/IMS/pages/1204585768/Max+min+values+in+QTI+interactions

### CODE REVIEW

**Comment by habemuscode** [comment]
Related PR: https://github.com/oat-sa/extension-tao-itemqti/pull/1666

**Review by julien-sebire — APPROVED** [review]
Looks good according to spec.
Can't test at the moment.

File: views/js/qtiCreator/widgets/component/minMax/minMax.js [hunk 1]
```diff
@@ -348,7 +348,7 @@ define([
                         if(fromField === fields.min && minValue > maxValue){
                             this.setMaxValue(minValue);
                         }
-                    } else if (minValue === 0 && maxValue > 0) {
+                    } else if ((minValue === 0 && maxValue > 0) && (document.querySelector('.edit-active > .qti-orderInteraction') || document.querySelector('.edit-active > .qti-graphicOrderInteraction'))) {
                         this.enableField(fields.min, 1);
                         this.$component[0].querySelector('input[name=minChoices-toggler]').checked = true;
```
Comment (bziondik) [inline views/js/qtiCreator/widgets/component/minMax/minMax.js:353]:
```suggestion
                       controls.min.toggler.prop('checked', true);
```

**Review by bziondik — COMMENTED** [review]
We already have controls

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**